tests/e2e/memgraph.py

```diff
@@ -130,9 +130,14 @@ def is_running(self):
     def stop(self):
         if not self.is_running():
             return
-        self.proc_mg.terminate()
-        code = self.proc_mg.wait()
-        assert code == 0, "The Memgraph process exited with non-zero!"
+
+        pid = self.proc_mg.pid
+        try:
+            os.kill(pid, 15)  # 15 is the signal number for SIGTERM
```

Comment:
Why this change?
Popen.terminate() - Stop the child. On POSIX OSs the method sends SIGTERM to the child.

Reply:
Yes, but there was some issue with this method. It reported that the memgraph didn't properly close, although from logs and by checking manually everything was fine. I checked for different ways to close the memgraph, and this one works. I couldn't see any issue, so I presumed the method was the problem.